client/src/components/MapViewEntry.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import React from 'react';
+import { Divider, Row, Col } from 'antd';
+import { CaretRightFilled } from '@ant-design/icons';
+
+const MapViewEntry = (props: Props) => {
+  const { selected } = props;
+  const color = selected ? '#1890FF' : '#D9D9D9';
+
+  return (
+    <>
+      <Row style={{ width: 300 }} align="middle">
+        <Col span={16}>
+          <Row>
+            <b>Crisis Nursery</b>
+          </Row>
+          <Row justify="space-between">
+            <Col>Urbana</Col>
+            <Col>•</Col>
+            <Col>1309 West Hill Street</Col>
+          </Row>
+          <Row justify="space-between">
+            <Col>$</Col>
+            <Col>•</Col>
+            <Col>English, Spanish, French, Chinese</Col>
+          </Row>
```

Comment:
I want to say this should be aligned left - but maybe that's more of a question for @Lam7150 
Also, in case the text overflows on both of these, I think we'll want to use [this](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) with the ellipsis

Reply:
yes left-align this, since more this may overflow depending on the screen size